demo/api.md

```diff
@@ -78,6 +38,86 @@
 
 </auro-accordion>
 
+#### Dynamic Menu
+
+This example demonstrates a data driven combobox. The data is used to populate the `auro-menu`. In 
```

Comment:
We need to add context to this menu here. I have no idea what to even start searching for? I just randomly typed a letter hoping to get a return.

Reply:
This was deleted because I sometimes delete the contents of a demo page when I am testing my code against one single example so I don't have to scroll up and down every page refresh. I am adding it back with a patch.